app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [REQUIRED] Hello team, kindly enable `:confirmable` in the user model.  Confirmable is responsible to verify if an account is already confirmed to sign in and sending emails with confirmation instructions.